docs/model_library/operational_water_management/index.rst

```diff
@@ -748,7 +752,17 @@ For each treatment site, for each time period, the treatment cost is equal to al
     \textcolor{red}{C_{r,t}^{Treatment}} = (∑_{(l,l)∈{NRA,SRA}}\textcolor{red}{F_{l,l,t}^{Piped}}+∑_{(l,l)∈{PRT,CRT}}\textcolor{red}{F_{l,l,t}^{Trucked}})⋅ \textcolor{green}{π_{r}^{Treatment}}
 
     \textcolor{red}{C^{TotalTreatment}} = ∑_{∀t∈T}∑_{r∈R}\textcolor{red}{C_{r,t}^{Treatment}}
+    
+
+**Unused Treated Water Cost:** ∀r ∈ R, t ∈ T
+
+For each treatment site, for each time period, the unused treated water cost is equal to all water not used after treating multiplied by the highest disposal cost. The total unused treated water cost is the sum of unused treated water costs over all time periods and all treatment sites.
+
+.. math::
+
+    \textcolor{red}{C_{r,t}^{UnusedTreatedWater}} = \textcolor{red}{F_{r,t}^{UnusedTreatedWater}}⋅ Max_{∀k∈K}(\textcolor{green}{π_{k}^{Disposal}})
```

Comment:
If you write `\max_` here instead of `Max_`, will the "max" function be displayed in normal text instead of italics? I think it would look better that way, and that's how you would write it in LaTeX. This comment also applies to the equivalent instance in strategic_water_managmement\index.rst.

Reply:
resolved by the recent documentation update PR